### DISCUSSION TITLE
Please help me to collect nice Anime

### DISCUSSION THREAD

**Discussion by satokibi44** [NAIST-SE/AttractingContributors#15] [General]
https://github.com/satokibi44/AttractingContributors-Anime

**Comment by tattiyaS** [comment]
add an anime!

**Reply by satokibi44** [reply]
Thank you!
I also sent PR to you!

**Comment by haxhimitsu** [comment]
Thanks your merge:)

**Reply by satokibi44** [reply]
Thank you!
I already sent PR to you!

**Comment by nosfill** [comment]
add an anime

**Reply by satokibi44** [reply]
Thank you!
I already sent PR to you!

**Comment by wowry** [comment]
Added!

**Reply by satokibi44** [reply]
Thank you!
I already sent PR to you!

**Comment by tkmaeda** [comment]
Just sent you a PR!

**Reply by satokibi44** [reply]
Thank you!
I also sent PR to you!

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors

**Reply by satokibi44** [reply]
thank you .
I sent PR.

**Comment by vittunyuta-aom** [comment]
added please help me https://github.com/aommoaGitHub/SuperFavoriteSongs

**Reply by satokibi44** [reply]
thank you .
I sent PR.
